frontend/task/task.scss

```diff
@@ -309,6 +309,108 @@ body {
       }
     }
   }
+
+  .task-recorder-controls {
+    display: flex;
+    align-items: center;
+    background: #4a4a4a;
+
+    .controls-recorder {
+      display: flex;
+      align-items: center;
+
+      .bp3-button {
+        background: transparent !important;
+        box-shadow: none !important;
+        border-right: solid 1px black;
+        padding: 10px 15px;
+        &:hover {
+          background-clip: padding-box;
+          background-color: #848484 !important;
+        }
+        &[disabled] {
+          opacity: 0.5;
+        }
+        .bp3-icon {
+          color: #fff !important;
+          svg {
+            width: 18px;
```

Comment:
The buttons are a bit small, which could make it complicated for touch users, especially for that record button right in the corner of the screen.

I would add 10px : `width: 28px; height: 28px;` looks good to me.